srcpkgs/nushell/template

```diff
@@ -1,22 +1,28 @@
 # Template file for 'nushell'
 pkgname=nushell
-version=0.21.0
+version=0.26.0
 revision=1
-# raw-cpuid does not support other archs
-# https://github.com/nushell/nushell/issues/485
-# remove when the issue is fixed (should fix all archs)
-archs="x86_64* i686*"
-wrksrc="nu-${version}"
 build_style=cargo
-hostmakedepends="pkg-config"
-makedepends="libressl-devel"
+hostmakedepends="pkg-config libgit2-devel"
+makedepends="libressl-devel libgit2-devel"
 short_desc="Modern shell for the GitHub era"
 maintainer="Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
 license="MIT"
 homepage="https://www.nushell.sh/"
 changelog="https://www.nushell.sh/blog/"
-distfiles="https://static.crates.io/crates/nu/nu-${version}.crate"
-checksum=44b0dc4127433373d6ce1b5ba50c56780f23b861654fce3b48d5b599758e772c
+distfiles="https://github.com/nushell/nushell/archive/${version}.tar.gz"
+checksum=66fbfe1297997a3f6b2181cd723816150ad2453527c7cab6c83a9c67b9af2478
+register_shell="/bin/nu"
+
+case "$XBPS_TARGET_MACHINE" in
+	x86_64* | i686*)
+		configure_args="--features=extra"
+		makedepends+=" libX11-devel libxcb-devel" ;;
+esac
```

Comment:
Is this definitely because of arch and not due to being crossbuilds?

Reply:
Yes. You can check the issue I linked above.
The "extra" feature set has `bson` that will not build on arm, and crate `xcb` needs `python` as dependency.
Crossbuilds work, but it will complicate the template (like previous commit)